mars-core/src/test/java/com/mars/core/util/FileUtilTests.java

```diff
@@ -0,0 +1,120 @@
+package com.mars.core.util;
+
+import com.diffblue.deeptestutils.mock.DTUMemberMatcher;
+import com.alibaba.fastjson.JSON;
+import org.ho.yaml.Yaml;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.Rule;
+import org.junit.runner.RunWith;
+import org.junit.rules.ExpectedException;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import java.awt.image.BufferedImage;
+import java.io.*;
```

Comment:
is the wildcard necessary here?

Reply:
removed.. i didn't see that.. it must have been an auto import